libs/output-mapping/src/Writer/Table/TableDefinition/TableDefinitionFactory.php

```diff
@@ -23,7 +23,7 @@ public function createTableDefinition(string $tableName, array $primaryKeys, arr
         $tableDefinition->setPrimaryKeysNames($primaryKeys);
         foreach ($columnMetadata as $columnName => $metadata) {
             $tableDefinition->addColumn(
-                $columnName,
+                (string) $columnName,
```

Comment:
Jak to máme jako pole, tak je to peklo. Přetypováváme to na X místech.

Založil jsem issue, abysme to předělali https://keboola.atlassian.net/browse/PST-1962

Reply:
Tohle by mělo být poslední neošetřený místo. Jinde co jsem se koukal to ošetřeno je.